generator/internal/genclient/translator/openapi/openapi_test.go

```diff
@@ -44,23 +44,28 @@ func TestMakeAPI(t *testing.T) {
 		Documentation: "A resource that represents a Google Cloud location.",
 		Name:          "Location",
 		Fields: []*genclient.Field{
-			{Name: "name",
+			{
+				Name:          "name",
 				Documentation: "Resource name for the location, which may vary between implementations.",
 				Typez:         genclient.STRING_TYPE,
 				Optional:      true},
```

Comment:
ditto re: indenting between `true` and `}` here and below

Reply:
Done.